draft-ietf-dprive-dnsoquic.md

```diff
@@ -569,11 +554,27 @@ connections. For example:
 
 Servers and Clients manage flow control as specified in QUIC.
 
-Servers MAY use the "maximum stream ID" option of the QUIC transport to limit
+Servers MAY use the "maximum stream ID" parameter of the QUIC transport to limit
 the number of streams opened by the client. This mechanism will effectively
 limit the number of DNS queries that a client can send on a single DoQ
-connection.
+connection. The initial value of this parameter is specified by the transport
+parameter `initial_max_streams_bidi`. For better performance, it is RECOMMENDED
+that servers chose a sufficiently large value for this parameter.
```

Comment:
This last sentence depends on a shared understanding of "sufficient" in this context, which is pretty fraught without more guidance.